spec/jobs/clean_all_job_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
```

Comment:
This should not be required because its part of the .rspec configuration file: https://github.com/errbit/errbit/blob/v0.6.1/.rspec#L2